csrc/root_domain_map.h

```diff
@@ -531,7 +543,7 @@ class TORCH_CUDA_CU_API ComputeAtRootDomainMapBuilder
 //! domains with non-broadcast domains.
 class TORCH_CUDA_CU_API ExactRootDomainMap : public RootDomainMap {
  public:
-  ExactRootDomainMap(Fusion* fusion);
+  ExactRootDomainMap(Fusion* fusion, bool map_symbolic = false);
```

Comment:
When do we need to map symbolic domains? Is it actually used?

Reply:
This is done in concretization inside `propagateFromProducerToConsumer`
```c++
  for (auto producer : ir_utils::filterByType<TensorView>(def->inputs())) {
    PairwiseRootDomainMap root_map(producer, consumer);
    // We map symbolic domains here regardless of whether their extents match.
    // This is safe because we are propagating from a producer which should have
    // already been concretized. The consumer might have a different extent
    // which will be equivalent to (but not necessarily sameAs) the producer's,
    // and we just want to use its IterType to concretize the consumer ID.
    root_map.mapSymbolic(true);
```

Reply:
That's `PairwiseRootDomainMap`. Do we need `map_symbolic` of `ExactRootDomainMap`?